### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'UploadedTo'

### ISSUE THREAD

**Issue by Frozen-byte** [pyload/pyload#1840]
I am on:
`HEAD is now at fa5e570 [Plugin] Fix _log + code cosmetics`
Simple Uploaded.to Link.

Puts following Error:
```
22.09.2015 23:06:57 ERROR     Error importing UploadedTo: invalid syntax (Hoster.py, line 212)
22.09.2015 23:06:57 CRITICAL  'NoneType' object has no attribute 'UploadedTo'
Traceback (most recent call last):
  File "/root/pyload/module/ThreadManager.py", line 276, in assignJob
    job.initPlugin()
  File "/root/pyload/module/utils.py", line 165, in new
    return func(*args)
  File "/root/pyload/module/PyFile.py", line 102, in initPlugin
    self.pluginclass = getattr(self.pluginmodule, self.m.core.pluginManager.getPluginName(self.pluginname))
AttributeError: 'NoneType' object has no attribute 'UploadedTo'
```

Any Ideas?

**Comment by obli88** [comment]
pyload/module/plugins/internal/Hoster.py line 211:
            self.log_info(_("Requesting line reconnection...")
has to be:
            self.log_info(_("Requesting line reconnection..."))
There is a closing bracket missing.

**Comment by denkbrett** [comment]
after i do this change inst Hoster.py  i have this in debug rebort form Uploaded:

 Traceback (most recent call last):
  File "/usr/share/pyload/module/PluginThread.py", line 187, in run
    pyfile.plugin.preprocessing(self)
  File "/usr/share/pyload/module/plugins/internal/Hoster.py", line 184, in preprocessing
    return self._process(*args, **kwargs)
  File "/usr/share/pyload/module/plugins/internal/Hoster.py", line 168, in _process
    self.load_account()  #@TODO: Move to PluginThread in 0.4.10
  File "/usr/share/pyload/module/plugins/internal/Hoster.py", line 195, in load_account
    if not self.account.user:  #@TODO: Move to `Account` in 0.4.10
AttributeError: 'UploadedTo' object has no attribute 'user'

**Comment by nafex** [comment]
fixed with #1843
